src/figuresClasses.ts

```diff
@@ -1,11 +1,93 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | `circle` | `rectangle`;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  readonly shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sorted: number[] = [a, b, c].sort((el1, el2) => el2 - el1);
```

Comment:
Not required - but I would not use `sort` for such case

I would suggest looking at `Math.max`